tests/suites/test_suite_ssl.function

```diff
@@ -3791,17 +3838,24 @@ exit:
 /* END_CASE */
 
 /* BEGIN_CASE depends_on:MBEDTLS_X509_CRT_PARSE_C:!MBEDTLS_USE_PSA_CRYPTO:MBEDTLS_PKCS1_V15:MBEDTLS_RSA_C:MBEDTLS_ECP_DP_SECP384R1_ENABLED:MBEDTLS_ENTROPY_C:MBEDTLS_CTR_DRBG_C */
-void handshake_version( int version, int dtls )
+void handshake_version( int dtls, int client_min_version, int client_max_version,
+                        int server_min_version, int server_max_version, int expected_version )
```

Comment:
The position of 'expected_version' is a bit weird in this rendering, over 80 characters?

Reply:
It wasn't, but the below fix made it so, so its on the next line now anyway.